plugins/scaffolder/src/api.ts

```diff
@@ -70,7 +70,11 @@ export interface ScaffolderApi {
    * @param templateName - Name of the Template entity for the scaffolder to use. New project is going to be created out of this template.
    * @param values - Parameters for the template, e.g. name, description
    */
-  scaffold(templateName: string, values: Record<string, any>): Promise<string>;
+  scaffold(
+    templateName: string,
+    values: Record<string, any>,
+    secrets?: JsonObject,
```

Comment:
need to add a new `@param` above :)

Reply:
Also looks like we might want to unify the types of this and the `values`.

Can secrets really be anything but string valued btw? Is `Record<string, string>` better here?